riscv/encoding.h

```diff
@@ -38,8 +38,6 @@
 
 #define MSTATUSH_SBE        0x00000010
 #define MSTATUSH_MBE        0x00000020
-#define MSTATUSH_GVA        0x00000040
-#define MSTATUSH_MPV        0x00000080
```

Comment:
It seems this file has been hand-edited recently, instead of generating it from riscv-opcodes. (I know I am guilty of this.) There are several changes in this commit that seem to be reverting recent hand changes to this file.

It would be best to update riscv-opcodes as needed first, auto-generate a clean version of this encoding.h that matches what had been hand changes before, then make any changes needed for Krypto as a separate commit.

Especially the removal of `uret` and related CSRs should be a separate commit or even separate PR.

Reply:
It was already a separate PR and was merged before this one.

Reply:
I guess you're talking about #847? That removed one line in disasm.cc.

If it had already been removed in a separate PR then it wouldn't be showing up here in this diff.

Reply:
Andrew requested me to file a separate PR while he was reviewing this, so that's how it was gone. You're currently commenting on a squashed (outdated) commit of an autogenerated header file.

Reply:
.. which I probably shouldn't have included in the original commit. I wasn't expecting to have to split and squash it into things that go into separate repos *sigh*

Reply:
I'm commenting on 236fd642967bddeece5312b6a8fa52c5ab693b38 which is in your latest mjosaarinen:master branch as of this second.

Reply:
Ok.. the PR is in for these bits. https://github.com/riscv/riscv-opcodes/pull/86